src/styles/theme.ts

```diff
@@ -17,12 +17,12 @@ const theme = {
       hover: "#000411",
     },
     grey: {
-      veryLight: "#e1e1f1",
-      light: "#f3f3f8",
+      veryLight: "#f3f3f8",
```

Comment:
I would remove all the colors that we are not using in the bridge by the moment. Apart from that I would try to put some order in this file.